.github/workflows/tests.yml

```diff
@@ -27,25 +28,51 @@ jobs:
       - name: Tests Maven cache
         uses: actions/cache@v3
         with:
-          path: |
-            tests/petclinic-test/.m2
-            tests/pdfbox/.m2
+          path: ${{ runner.temp }}/.m2
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-
 
+      - name: Tests PDFBox
+        uses: actions/cache@v3
+        with:
+          path: tests/pdfbox/pdfbox-2.0.27
+          key: tests-pdfbox-pdfbox-2.0.27
+          restore-keys: |
+            tests-pdfbox-pdfbox-2.0.27
+
+      - name: Tests Petclinic
+        uses: actions/cache@v3
+        with:
+          path: tests/petclinic-test/spring-petclinic
+          key: tests-petclinic-test-spring-petclinic
+          restore-keys: |
+            tests-petclinic-test-spring-petclinic
+
       - name: Build the chiselled-jre image
         run: |
           docker build \
           -t ${{ env.image-name }} \
+          --build-arg UID=$(id -u ${USER}) \
+          --build-arg GID=$(id -g ${USER}) \
           -f chiselled-jre/Dockerfile.${{ matrix.ubuntu-release }} \
           chiselled-jre
 
+      - name: Build the JDK docker image
+        run: |
+          docker build \
+          -t ${{ env.build-image-name }} \
```

Comment:
this one should probably not be called "chiselled"

Reply:
Done. Also noticed that I had DOCKER_OPTIONS in one script and DOCKER_OPTS in another. Aligned them.